### PR TITLE
Add 8 v2v cases converting guest from xen host

### PR DESCRIPTION
**Pull request by chloerh**

### CODE REVIEW

**Comment by chloerh** [comment]
test result for 8 cases:

> JOB ID     : b0e2ff079d09a7adf140e4c8442dc704e0e1508d
> JOB LOG    : /root/avocado/job-results/job-2016-05-13T17.35-b0e2ff0/job.log
> TESTS      : 8
>  (1/8) type_specific.local.specific_xen.positive_test.libvirt.pool_uuid.output_mode.libvirt: FAIL (248.74 s)
>  (2/8) type_specific.local.specific_xen.positive_test.libvirt.windows.rhev_file.output_mode.libvirt: FAIL (482.77 s)
>  (3/8) type_specific.local.specific_xen.positive_test.rhev.multiconsole.output_mode.rhev: PASS (758.52 s)
>  (4/8) type_specific.local.specific_xen.positive_test.rhev.console_xvc0.output_mode.rhev: PASS (764.47 s)
>  (5/8) type_specific.local.specific_xen.positive_test.rhev.xvda_disk.output_mode.rhev: PASS (803.72 s)
>  (6/8) type_specific.local.specific_xen.positive_test.rhev.xvda_guest.output_mode.rhev: PASS (801.64 s)
>  (7/8) type_specific.local.specific_xen.positive_test.rhev.guest_uuid.output_mode.rhev: PASS (742.08 s)
>  (8/8) type_specific.local.specific_xen.positive_test.rhev.windows.default_install.output_mode.rhev: PASS (1232.68 s)
> RESULTS    : PASS 6 | ERROR 0 | FAIL 2 | SKIP 0 | WARN 0 | INTERRUPT 0
> JOB HTML   : /root/avocado/job-results/job-2016-05-13T17.35-b0e2ff0/html/results.html
> TIME       : 5834.62 s

**Comment by chloerh** [comment]
for 2 failed cases please refer to 
[Bug 1141631 - [RFE] virt-v2v should support convert a guest to a dir-pool with using pool's uuid](https://bugzilla.redhat.com/show_bug.cgi?id=1141631)
[Bug 1161019 - Remove rhev related files(rhsrvany.exe and rhev-apt.exe) after conversion of windows guests to kvm](https://bugzilla.redhat.com/show_bug.cgi?id=1161019)

File: v2v/tests/cfg/specific_xen.cfg [hunk 1]
```diff
@@ -0,0 +1,118 @@
+- specific_xen:
```
Comment (will-Do) [inline v2v/tests/cfg/specific_xen.cfg]: case name

File: v2v/tests/src/specific_xen.py [hunk 2]
```diff
@@ -0,0 +1,213 @@
...
+        vmcheck = utils_v2v.VMCheck(test, params, env)
+        vmcheck.create_session()
+        grub_file = vmcheck.get_grub_path()
+        if not grub_file:
+            raise exceptions.TestError('Not found grub file')
+        content = vmcheck.session.cmd('cat %s' % grub_file)
+        if check == 'console_xvc0':
+            if 'console=xvc0' in content:
+                raise exceptions.TestFail('"console=xvc0" still exists')
+        vmcheck.session.close()
```
Comment (will-Do) [inline v2v/tests/src/specific_xen.py]: Better to use ```try...finally``` so you can always close the session

**Comment by will-Do** [comment]
OK, merge this pr

**Pull request merged**